features/step_definitions/web_steps.rb

```diff
@@ -1,3 +1,3 @@
 When(/^I click the (.*) "(.*)"$/) do |elem, text|
-  find(elem).find('*', text: text).click
-end
\ No newline at end of file
+  all(elem, text: text, count: 1)[0].click
```

Comment:
there is an issue here. simply picking the first element is not a viable solution. perhaps try `find(elem, text: text).click`?